File: testbed_ruby/test/statemachine/EmptyStateTest.php
require 'test/unit'
require './src-gen-umple/student_'
require './src-gen-umple/mentor_'
require './src-gen-umple/program_'

module CruiseTest
class EmptyStateTest < Test::Unit::TestCase

    
  function test_getStateMachine
    course = new CourseA
    assert_equal(nil,course.getStatus)
  end
end

File: testbed_python/test/statemachine/empty_state_test.py
import unittest

# Test classes are imported dynamically
# Must be generated into ImportModules' namespace then imported
from ImportModules import importModules

importModules(["CourseA", ], ["statemachine", "test"])
from ImportModules import *


class EmptyStateTest(unittest.TestCase):
    def test_getStateMachine(self):
        course = CourseA.CourseA()
        self.assertEquals(None, course.getStatus())
